x-pack/plugins/security_solution/public/entity_analytics/translations.ts

```diff
@@ -244,3 +244,18 @@ export const UPDATE_PANEL_GO_TO_DISMISS = i18n.translate(
     defaultMessage: 'Dismiss',
   }
 );
+
+export const MAX_SPACE_PANEL_TITLE = i18n.translate(
+  'xpack.securitySolution.riskScore.maxSpacePanel.title',
+  {
+    defaultMessage: 'Sorry, you reached the maximum amount of risk scoring.',
```

Comment:
I defer to @SourinPaul and @craig-abbott on exact copy here, but "amount of risk scoring" is odd.